plugins/ruby-foreman-tasks/debian/control

```diff
@@ -2,7 +2,7 @@ Source: ruby-foreman-tasks
 Section: ruby
 Priority: extra
 Maintainer: Michael Moll <mmoll@mmoll.at>
-Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.7)
+Build-Depends: debhelper, foreman (>= 1.8.0~rc1), foreman-sqlite3 (>= 1.8.0~rc1), ruby-foreman-dynflow (>= 0.7.9)
```

Comment:
As we reflect the dependencies from the gems in the packages, please leave that as ```ruby-foreman-dynflow (>= 0.7.7)``` for now and when a new foreman-tasks gem is relased with ```dynflow >= 0.7.9```, change it in the foreman-tasks packaging PR.